### DISCUSSION TITLE
XY Chart - multiple yAxis configuration

### DISCUSSION THREAD

**Discussion by melloc01** [airbnb/visx#1432] [Q&A] [answered]
Hi,

is there a way to configure multiple `<Yaxis />` and use each series-related asset on a `<LineSeries />` component?

**Comment by williaster** [comment, marked as answer]
Hi @melloc01 , this is not an issue and thus I converted it to a discussion.

This duplicates https://github.com/airbnb/visx/issues/1189 and https://github.com/airbnb/visx/issues/1318

**tl;dr** dual y-axis charts are not recommended because they are difficult to interpret and can be misleading. If you choose this type of chart you'll need to use the lower-level visx packages to create it yourself with `@visx/axis` + `@visx/scale`

**Reply by melloc01** [reply]
Alright, thanks @williaster 

Since I am able to create multiple charts it made sense to me that xy-chart package could have this feature too, but anyways I'd need more custom stuff so I'm already on the low level packages now

Thanks again :wave:
